src/utils/wallet.js

```diff
@@ -689,7 +689,16 @@ class Wallet {
     async setupRecoveryMessageNewAccount(accountId, method, securityCode, fundingOptions, recoverySeedPhrase) {
         const { secretKey } = parseSeedPhrase(recoverySeedPhrase)
         const recoveryKeyPair = KeyPair.fromString(secretKey)
-        await this.validateSecurityCode(accountId, method, securityCode);
+        
+        try {
+            await this.validateSecurityCode(accountId, method, securityCode);
+        } catch (error) {
+            if (error.toString().includes('Error: Invalid code')) {
+                throw new WalletError(error, 'setupRecoveryMessageNewAccount.invalidCode')
```

Comment:
Isn’t first parameter supposed to be a string vs error?

Reply:
@vgrichina I refactored it, `validateSecurityCode` was already handling this error, just an alert was not showing